config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
```

Comment:
- [ ] Nice job on adding the devise however, when the local host start I should see the login page. right now the page is the user page I see. make sure the landing page in the routes is the register or login.

![Capture web_4-4-2022_94412_localhost](https://user-images.githubusercontent.com/61562626/161518968-8413e831-df54-4b56-a927-3cc60fd6341b.jpeg)

Reply:
Thanks, @dasileker  for the feedback and Ramadan Mubarak